OutOfSchool/OutOfSchool.WebApi/Controllers/V1/FavoriteController.cs

```diff
@@ -79,7 +79,7 @@ public async Task<IActionResult> GetById(long id)
     [HttpGet]
     public async Task<IActionResult> GetAllByUser()
     {
-        string userId = User.FindFirst("sub")?.Value;
+        string userId = User.FindFirst("http://schemas.xmlsoap.org/ws/2005/05/identity/claims/nameidentifier")?.Value;
 
```

Comment:
Better to use ClaimTypes.NameIdentifier

Reply:
The another way - IdentityResourceClaimsTypes.Sub

Reply:
I don't believe you 😭

Reply:
Yes, it works.  :)